lib/get-html.ts

```diff
@@ -28,6 +28,7 @@ export const getHtml = async (
   const $ = cheerio.load(body);
 
   // get only the HTML we care about to avoid unnecessary cache updates
+  $('[data-testid="callout"]').remove();
```

Comment:
oh that's awesome! (I'd not looked into this, so not surprised it was that easy 😅)